### PR TITLE
Blaze Manage Campaigns: Add dark mode support to Blaze Campaign card

### PR DESCRIPTION
**Pull request by kean**
Related Issue: #20745

To test:
- In `DashboardCard.swift`, replace `DashboardBlazeCardCell.self` with `DashboardBlazeCampaignCardCell.self`
- Open Dashboard with Blaze approved/enabled
- Update status in `DashboardBlazeCampaignStatusView.swift` (it will be easier to test when the backend is integrated and can be mocked dynamically)

## Regression Notes
1. Potential unintended areas of impact: n/a (only affects the new card)
2. What I did to test those areas of impact (or what existing automated tests I relied on): manual testing
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Blaze/DashboardBlazeCampaignCardCell.swift [hunk 1]
```diff
@@ -87,6 +87,12 @@ final class DashboardBlazeCampaignCardCell: DashboardCollectionViewCell {
         BlazeFlowCoordinator.presentBlaze(in: presentingViewController, source: .dashboardCard, blog: blog)
     }
 
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        createCampaignButton.setTitleColor(UIColor.primary, for: .normal)
+    }
+
```
Comment (momo-ozawa) [inline WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Blaze/DashboardBlazeCampaignCardCell.swift]: Do we need this?

**Review by momo-ozawa — APPROVED** [review]
LGTM, left 1 comment

File: WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Blaze/DashboardBlazeCampaignCardCell.swift (hunk 1 above)
Comment (kean) [reply WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Blaze/DashboardBlazeCampaignCardCell.swift]: It looked like it wasn't updating and was too dark yesterday evening. I double-checked with a color meter, and no, it does update, and it's not needed. Removed. Something off with my color perception 😁

Comment (momo-ozawa) [reply WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Blaze/DashboardBlazeCampaignCardCell.swift]: 🙈

**Pull request merged**